src/algebraic_geometry/presheafed_space.lean

```diff
@@ -67,7 +76,7 @@ end
 
 def id (X : PresheafedSpace C) : hom X X :=
 { base := 𝟙 (X : Top.{v}),
-  c := ((functor.left_unitor _).inv) ≫ (whisker_right (nat_trans.op (opens.map_id (X.to_Top)).hom) _) }
+  c := ((functor.left_unitor _).inv) ≫ (whisker_right (nat_trans.op (opens.map_id (X.carrier)).hom) _) }
 
 def comp (X Y Z : PresheafedSpace C) (α : hom X Y) (β : hom Y Z) : hom X Z :=
```

Comment:
Could you take care of [the final nolints](https://github.com/leanprover-community/mathlib/blob/master/scripts/nolints.txt#L30) while you're editing the file?

Reply:
Got it!